tests/improver-ensemble-calibration/07-current_forecast_percentiles.bats

```diff
@@ -40,7 +40,7 @@
       'gaussian' "$IMPROVER_ACC_TEST_DIR/ensemble-calibration/percentiles/input.nc" \
       "$IMPROVER_ACC_TEST_DIR/ensemble-calibration/gaussian/history/*.nc" \
       "$IMPROVER_ACC_TEST_DIR/ensemble-calibration/gaussian/truth/*.nc" \
-      "$TEST_DIR/output.nc" --num_realizations=18 --max_iterations 200
```

Comment:
Why this change?

Reply:
The `max_iterations` argument was previously included for stability reasons, however, is no longer required.